.github/workflows/push.yml

```diff
@@ -60,16 +56,16 @@ jobs:
           popd
         shell: bash
 
-      - name: Publish npm package to Artifactory
+      - name: Publish npm package
         run: |
-          export VERSION="${{needs.release.outputs.version}}"
+          export VERSION="${{ needs.release.outputs.version }}"
           echo "Publishing package with version $VERSION"
-          pushd javascript
```

Comment:
Doesn't this deletion break the workflow?

Reply:
Good call. Thank you for catching this!